src/Traits/ConvertsEntityToWebsite.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Traits;
+
+use Hyn\Tenancy\Contracts\Hostname;
+use Hyn\Tenancy\Contracts\Website;
+
+trait ConvertsEntityToWebsite
+{
+    /**
+     * @param $to
+     * @return Hostname|Website|null
```

Comment:
Should be `Website|null`, no `Hostname`.

Reply:
It can also be a Hostname, hence the `instanceof` test a little down below.